_events/sr2020/virtual-competition-knockouts.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Virtual Competition Knockouts and Final
+date: 2020-07-25 15:00:00
+layout: event
+type: competition
+location: Online
+---
+
+The Student Robotics Competition marks the ultimate stage of SR2020, showcasing the months of preparation. Our teams will work hard to push their virtual robots to the limit, competing against each other, and evolving their strategies to gain the upper hand.
+
+Code submission will be due at 10am. Our live-steam will start at 3pm.
+
+[Watch live](https://youtu.be/xBPVqsb_Ydk)
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/xBPVqsb_Ydk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Note: I'm assuming we'll do the usual thing of replacing the homepage with the competition-website homepage, so the stream will be on the homepage too.